tests/PageObjects/SongListPage.ts

```diff
@@ -47,7 +47,26 @@ export class SongListPagePO {
   public getPlaylistElement(name: string) {
     return this.page.getByTestId(`playlist-${name}`);
   }
+
   public async goToPlaylist(name: string) {
     await this.getPlaylistElement(name).click();
   }
+
+  public async expectPlaylistToBeVisibleAsSelected(name: string) {
```

Comment:
`expectPlaylistToBeVisible` maybe?

Reply:
all playlist are visible, so maybe: `expectPlaylistToBeSelected` will be better?

Reply:
sure :)